docs/book/src/advanced/traits.md

```diff
@@ -9,7 +9,7 @@ Let's take a look at some code:
 ```sway
 trait Compare {
     fn equals(self, b: Self) -> bool;
-} {
+
```

Comment:
This is correct here - `equals()` is part of the interface block whereas `not_equals()` is part of the implementation block.